tests/MyGame/Example/Color.lua

```diff
@@ -4,7 +4,10 @@
 
 local Color = {
     Red = 1,
+-- /// \brief color Green
+-- /// Green is bit_flag with value (1u << 1)
     Green = 2,
+-- /// \brief color Blue (1u << 3)
```

Comment:
`///` should not be part of a Lua comment? and also misaligned?

Reply:
Lua needn't `///`. Misaligned also.
I will fix it.